README.md

```diff
@@ -315,16 +315,27 @@ If anything is not working, check the code in this repository (particularly the
 }
 ```
 
-## Committing our changes
+## Afraid of Commitment?
```

Comment:
Ouch! I understand the temptation to add your own style to things, but I almost rejected your PR outright because of this. Tutorials should have a consistent voice -- that of the author. There was nothing wrong with my heading, so your change is gratuitous and perhaps a bit arrogant? And you missed that headings are in sentence case, not title case. In short, not good to presume that the author simply doesn't know any better.